src/server/game/Movement/Spline/MovementPacketBuilder.cpp

```diff
@@ -98,7 +98,7 @@ namespace Movement
 
     void WriteLinearPath(const Spline<int32>& spline, ByteBuffer& data)
     {
-        uint32 last_idx = spline.getPointCount() - 3;
+        uint32 last_idx = spline.getPointCount() - 2;
```

Comment:
@Shauren Are any of those auto-generated points actually sent to the client?
Say I have a movespline like this in a sniff:
```
Position: X: 1607.088 Y: 802.5474 Z: 122.7526
(MovementMonsterSpline) (MovementSpline) Flags: 1048576 (Unknown5)
(MovementMonsterSpline) (MovementSpline) [0] Points: X: 1613.629 Y: 795.9061 Z: 121.8324
(MovementMonsterSpline) (MovementSpline) [0] WayPoints: X: 1609.108 Y: 799.9767 Z: 122.7925
(MovementMonsterSpline) (MovementSpline) [1] WayPoints: X: 1609.608 Y: 799.2267 Z: 122.7925
(MovementMonsterSpline) (MovementSpline) [2] WayPoints: X: 1612.608 Y: 796.9767 Z: 122.2925
```
which of these points should be passed to `MovebyPath` to replicate this packet in TC? Right now I pass the WayPoints in order, followed by the endpoint (the single Points entry). Is this correct?